worker-sys/src/types/durable_object/id.rs

```diff
@@ -14,3 +14,11 @@ extern "C" {
     #[wasm_bindgen(method, getter)]
     pub fn name(this: &DurableObjectId) -> Option<String>;
 }
+
+impl std::fmt::Debug for DurableObjectId {
```

Comment:
Perhaps align on `core`:

```suggestion
impl core::fmt::Debug for DurableObjectId {
```